### PR TITLE
Improve maintainability

### PR DESCRIPTION
**Pull request by joriba**
Improve SonarCloud's maintainability rating, mostly by applying Android Studio's recommended fixes.

### CODE REVIEW

File: app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -17,7 +17,6 @@
         <activity
             android:name=".MainActivity"
             android:exported="true"
-            android:label="@string/app_name"
```
Comment (yzueger) [inline app/src/main/AndroidManifest.xml:20]: Why is this removed?

**Review by KlaKalma — APPROVED** [review]
lgtm

File: app/src/main/AndroidManifest.xml (hunk 1 above)
Comment (tsogtb) [reply app/src/main/AndroidManifest.xml:20]: I think it was redundant.

Comment (joriba) [reply app/src/main/AndroidManifest.xml:20]: Android Studio told me it's redundant, so I removed it.

**Comment by joriba** [comment]
@yzueger I'm putting you in as a reviewer, because I had to merge a lot with your commits, just to make sure I left everything in the right place.

**Review by yzueger — APPROVED** [review]
lgtm, don't actually see that much merge conflict overlaps in the files changed
